backend/tasks/user.py

```diff
@@ -80,22 +93,32 @@ def put(self, username):
 
         # Check params
         if enable is None or username is None:
-            return make_response(json.dumps({"message": "Parameter 'enable' required"}), 400)
+            response = "Parameter 'enable' required"
+            AppLogger.log_message(
+                log_fn=logging.error, args=username, response=response
+            )
+            return make_response(json.dumps({"message": response}), 400)
 
         try:
             found_user = user.get_namespace_user(username)
             if found_user is None:
-                return make_response(
-                    json.dumps({"message": f"Could not find user {username}"}), 404
+                response = f"Could not find user {username}"
+                AppLogger.log_message(
```

Comment:
keep the `AppLogger.error(<message>)` as the abstraction. In the `AppLogger` class, do the logic of calling the `AppLogger.log_message(log_fn)` that way your code is transparent of the logger used

Reply:
Ok, I like that. I've pushed the changes